apps/ui-layout/components/website/home/hero-sec.tsx

```diff
@@ -102,37 +102,34 @@ const homeComponents = [
   },
   {
     id: 12,
-    link: '/components/dialog-card',
+    link: '/components/linear-card',
 
     componentSrc: <HomeAccordionModal />,
 
     className: 'lg:col-span-8  md:col-span-7 sm:col-span-6 col-span-12',
   },
   {
     id: 13,
-    link: '/components/products-cards',
+    link: '/components/product-cards',
 
     componentSrc: <HomeProductCard />,
     className: 'lg:col-span-4 md:col-span-5 sm:col-span-6 col-span-12',
   },
   {
     id: 14,
-    link: '/components/embla-carousel',
-
```

Comment:
why this was removed?
Is there any correct link?

Reply:
cause we've moved to  https://www.ui-layout.com/components/carousel